databases/week3/mealSharing.sql

```diff
@@ -0,0 +1,182 @@
+-- Now add a couple of different meals, reservations and reviews with different attributes. With those meals create the following queries
+-- Adding to meal
+INSERT INTO Meal (`title`, `description`,`location`, `when`,`max_reservations`, `price`,`created_date`) 
+VALUES ( 'Vegeterian pizza' , 'cauliflower, corn, bell pepper, mushroom, beans, ananas', 'Arhus','2021-02-02 12:00:00', 1 , 70, '2021-01-28');
+INSERT INTO Meal (`title`, `description`,`location`, `when`,`max_reservations`, `price`,`created_date`) 
+VALUES ( 'Cesar salad' , 'Chicken breast, lettuce, tomato, cucumber, corn, bread,special sauce', 'Hviding','2021-01-30 20:00:00', 4 , 150, '2021-01-28');
+INSERT INTO Meal (`title`, `description`,`location`, `when`,`max_reservations`, `price`,`created_date`) 
+VALUES ( 'Beans stew and rice' , 'rice , bean stew', 'Ribe','2021-01-31 18:30:00', 7 , 100, '2021-01-28');
+INSERT INTO Meal (`title`, `description`,`location`, `when`,`max_reservations`, `price`,`created_date`) 
+VALUES ( 'Lasagnia' , 'soya, cabbages, special sauce, corn', 'Alborg','2021-02-03 13:15:00', 5 , 135, '2021-01-28');
+
+-- Adding to Reservation
+INSERT INTO reservation (`number_of_guests`, `meal_id`,`created_date`, `contact_phonenumber`,`contact_name`, `contact_email`) 
+VALUES ( 1 , 5 ,'2021-01-28',  null ,'Christine', Null );
+INSERT INTO reservation (`number_of_guests`, `meal_id`,`created_date`, `contact_phonenumber`,`contact_name`, `contact_email`) 
+VALUES ( 3 , 4,'2021-01-29',  004511223344 ,'Lise', 'Lise@gmail.com' );
+INSERT INTO reservation (`number_of_guests`, `meal_id`,`created_date`, `contact_phonenumber`,`contact_name`, `contact_email`) 
+VALUES ( 2 , 3,'2021-01-31',  004599887766 ,'Yanal', 'Yanal@gmail.com' );
+
+-- Adding to review
+INSERT INTO review(`title`, `description`,`meal_id`, `stars`,`created_date`) 
+VALUES ('Not Good' ,'It was salty',  5 ,2, Null );
+INSERT INTO review(`title`, `description`,`meal_id`, `stars`,`created_date`)
+VALUES ( 'Good' ,NULL ,  7 ,5, '2021-01-23' );
+INSERT INTO review (`title`, `description`,`meal_id`, `stars`,`created_date`)
+VALUES ( 'Very good' ,'very tasty ',  4 ,5, '2021-02-12' );
+
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT * FROM meal
+WHERE price< 90;
+
+-- Get meals that still has available reservations
+SELECT * FROM meal
+WHERE max_reservations > (SELECT SUM(number_of_guests)
+        FROM reservation
+        WHERE meal_id = meal.id);
+            
+-- Get meals that partially match a title. Rød grød med will match the meal with the title Rød grød med fløde
+SELECT * FROM meal
+WHERE title LIKE '%Rød grød med%';
+
+-- Get meals that has been created between two dates
+SELECT * FROM meal
+WHERE created_date 
+BETWEEN  '2021-01-25' AND '2021-02-16';
+
+-- Get only specific number of meals fx return only 5 meals
+SELECT * FROM meal
+LIMIT 3;
+
+-- Get the meals that have good reviews
+SELECT meal.* FROM meal
+JOIN review ON meal.id= review.meal_id AND review.title='Good';
```

Comment:
Do we have a column **title** on table review? Try something like this if it works
```
SELECT  meal.id, meal.title, review.title, review.stars FROM meal, review
where review.stars>4  AND meal.id=review.meal_id;
```